release_notes.md

```diff
@@ -0,0 +1,6 @@
+# Fix Details
+
+- Removed `Scenario` module and SVT from the solution pack
```

Comment:
System View Template (SVT)

Reply:
Since its already a part of SOAR Framework Solution Pack, hence its not needed in this solution pack

Reply:
Made suggested changes in release notes